cfgov/core/views.py

```diff
@@ -123,3 +126,20 @@ def submit_comment(data):
                              })
 
     return response
+
+
+@csrf_exempt
```

Comment:
What would that look like? I've never actually coded with DRF.

It's not really a "restful" API, as it's a single endpoint.